.azure/azure-pipelines.yml

```diff
@@ -0,0 +1,32 @@
+# Azure pipeline used to trigger builds of the JPO CVDP project
+# This pipeline is primarily used for CI in the CDOT-CV fork
+
+trigger:
+  branches:
+    include:
+      - develop
```

Comment:
question: I see that in other repos we also trigger off of the dev/develop branch, why have we chosen that over master or some "release/*" nomenclature?

Reply:
I did not add this so I don't have insight. I can make it consistent.